packages/experimental/Avatar/src/titles.ts

```diff
@@ -101,6 +101,7 @@ export const titles = new Set<string>([
   'The Rt Hon Sir',
   'The Rt Hon Visc',
   'Viscount',
+  'van',
```

Comment:
Isn't van part of someone's name and not a title?

Reply:
I saw this case in PCX unit tests. That what I found "Van is a very common prefix in Dutch language surnames"